src/gui/mzroll/mainwindow.h

```diff
@@ -438,6 +438,11 @@ public Q_SLOTS:
      */
     void promptUpdate(QString version);
 
+    /**
+     * @brief updateTablePostAlignment Updating peak tables after alignment.
+     */
```

Comment:
Here's a better docstring: "Update the attributes of peaks for every group present in all current peak-tables. This method should be called after operations such as alignment, where the scan's RT changes from its original value."

Try to keep a balance between being descriptive and being precise, when writing documentation.